static/umap/theme.css

```diff
@@ -0,0 +1,76 @@
+/* **************************** */
+/*           navigation         */
+/* **************************** */
+nav h1 a {
+    background-image: url('./img/logo-marianne.gif');
+}
+.login-grid .login-datagouv {
+    background-image: url('../datagouv.png');
+}
+.branding {
+    width: 33%;
+}
+
+
+/* **************************** */
+/*             Forms            */
+/* **************************** */
+input[type="submit"], .button {
+    background-color: #2185d0;
+    color: #eeeeec;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-family: 'fira_sansregular';
```

Comment:
It doesn't add a lot to add fallbacks and in case the download of the font doesn't work you have decent defaults. By the way, using the same name as the default system name `"Fira Sans"` will avoid the download for Ubuntu users I think (to be verified).

The state of the art is to provide sane defaults and [then load the font using JS…](https://www.zachleat.com/web/comprehensive-webfonts/)